echo/echo-pubsub-aws/src/main/java/com/netflix/spinnaker/echo/pubsub/aws/AmazonMessageAcknowledger.java

```diff
@@ -53,17 +54,20 @@ public AmazonMessageAcknowledger(
 
   @Override
   public void ack() {
-    // Delete from queue
     try {
-      amazonSQS.deleteMessage(queueUrl, message.getReceiptHandle());
+      sqsClient.deleteMessage(
+          DeleteMessageRequest.builder()
+              .queueUrl(queueUrl)
+              .receiptHandle(message.receiptHandle())
+              .build());
       registry.counter(getProcessedMetricId(subscriptionName)).increment();
-    } catch (ReceiptHandleIsInvalidException e) {
```

Comment:
Looks like ReceiptHandleIsInvalidException exists in v2 as well.  It does seem more correct to catch the more general exception.